docs/source/getting_started/writing_code.ipynb

```diff
@@ -14,7 +14,7 @@
     ".. note::\n",
```

Comment:
They show up in the github preview but not the NB Review. The writing_code_overview.png shows up in both.

Reply:
@grallewellyn difference seems to be `../` added to the front of the partially working images, should probably just be _static/...

Reply:
Hmm I changed this because I moved all the clone_demo images from `source/getting_started/_static/` to `source/_static/` since I now also use the images in system_reference_guide 
If I delete the build files and rebuild locally the images are still showing up 
If I remove the `../` and delete the build files and rebuild locally the images don't show up 
@wildintellect

Reply:
When I build locally, the images are working for me

<img width="743" alt="Screenshot 2023-11-06 at 11 15 47 AM" src="https://github.com/MAAP-Project/maap-documentation/assets/1156193/c03905cd-f0f6-42db-b2bf-83e95f5bfbad">

Reply:
I am going to push this to the RtD site to test it out

Reply:
https://docs.maap-project.org/en/feature-update-git/getting_started/writing_code.html